client/my-sites/plans-features-main/index.jsx

```diff
@@ -36,8 +37,27 @@ import { plansLink } from 'lib/plans';
 import SegmentedControl from 'components/segmented-control';
 import SegmentedControlItem from 'components/segmented-control/item';
 import PaymentMethods from 'blocks/payment-methods';
+import HappychatButton from 'components/happychat/button';
+import HappychatConnection from 'components/happychat/connection-connected';
+import isHappychatAvailable from 'state/happychat/selectors/is-happychat-available';
+import { selectSiteId } from 'state/help/actions';
 
 class PlansFeaturesMain extends Component {
+	componentWillUpdate( nextProps ) {
+		/**
+		 * Happychat does not update with the selected site right now :(
```

Comment:
Hopefully some of these concerns will be addressed while we refactor the calypso-happychat integration by using the new `Automattic/happychat-client`